_posts/2022-12-06-integrating-auth0-with-react.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+font: serif
+title: Integrating Auth0 Authentication into React
+date: 2022-12-06
+highlight: monokai
+author: Chris Howard
+image: img/blog/integrating-auth0-with-react/banner.jpg
+tags: Auth0 react authentication typescript
+description: 
+---
+
+Authentication and Authorization is objectively the most important part of any application, So why is it so often left to the end of development cycles or implemented in a slap dash manner?
+
+It may be due to the difficulty in creating authentication systems and implementing them into our applications, but with the tools and services available to developers it has become easier and easier to create a robust, secure and easy to use system. So let's not wait and get started on integrating [Auth0](https://auth0.com/) into a new React application!
+
+<br />
+
+## Contents
+
+- [Why Auth0?](#why-auth0)
+- [Create Auth0 Application](#create-auth0-application)
+- [Integration into React](#integrate-with-react)
+    * [Login](#but-wait-how-do-users-login)
+    * [Logout](#great-now-how-about-logout)
+- [Conclusion](#conclusion)
+
+## Why Auth0?
```

Comment:
You don't introduce OAuth.

Do you expect the reader to understand about OAuth?  Is it pre-requisite knowleddge to this post?

You probably need a short sentence/introduction at least, with a link to other reference documentation for pople to learn more.

Reply:
Generally not a huge amount of knowledge regarding OAuth is strictly required for just the basic implementation, Was worried it would get too long

Will add a brief touch on it and a link